eeps/eep-0064.md

```diff
@@ -0,0 +1,354 @@
+    Author: Kiko Fernandez-Reyes <kiko(at)erlang(dot)org>,
+        Raimo Niskanen <raimo(at)erlang(dot)org>
+    Status: Draft
+    Type: Standards Track
+    Created: 07-Jun-2023
+    Erlang-Version: OTP-27
+    Post-History:
+****
+EEP 64: Multi-line Indented Verbatim Strings
+----
+
+Abstract
+========
+
+This EEP proposes the introduction of Multi-line Indented Verbatim strings,
+*MIV Strings* and defines their semantics.  The main benefit is to allow
+*multi-line* strings in an easy and useful, *indented*, way,
```

Comment:
```suggestion
*multi-line* strings in an easy, useful, and *indented* way,
```

Reply:
I'll go for `easy and useful (*indented*) way,`, which is more according to my intention